### PR TITLE
Use Thor and add `--version` command

### LINKED ISSUE #315: [CLI] Displaying the installed version via a flag
Running the command `neat -v` does not display the neat version number. I can run the same commands on bourbon and bitters to determine my version.

screen shot 2015-02-14 at 10 09 29 am

### PR DESCRIPTION
**Pull request by mehlah**
This moves to Thor, like Bitters and Bourbon, and add a `--version` command (was missing before, as described by #315)
Closes #315

### CODE REVIEW

**Comment by kaishin** [comment]
This looks good to me.

**Comment by mehlah** [comment]
Anything holding back this can I help with?

File: bin/neat [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
 
-require "neat"
+require File.dirname(__FILE__) + "/../lib/neat.rb"
```
Comment (teoljungberg) [inline bin/neat]: This change is seemingly unrelated. You can avoid this, and it's common to do something along these lines: `bundle exec ruby -I lib bin/neat`

Comment (tysongach) [inline bin/neat]: @teoljungberg Just built the gem locally and testing all of this with and without the above change, and it breaks without. This also aligns with the executable in Bourbon and Bitters, so I’m going to run with it.

**Comment by tysongach** [comment]
Merged as 4bef318692d2e0ba62112c6a2e7bc85e1e40f69d. Thanks, @mehlah!

**Pull request closed without merging**